### PR TITLE
[ISSUE #3492] DefaultMQPushConsumerImpl#adjustThreadPool() has no effect.

### PR DESCRIPTION
**Pull request by XiaoyiPeng**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

for issue #3492 

## Brief changelog

XX

## Verifying this change

XXXX

### CODE REVIEW

**Comment by XiaoyiPeng** [comment]
@dongeforever Take a look when you have time please, thank you very much.

File: client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java [hunk 1]
```diff
@@ -110,12 +110,14 @@ public void updateCorePoolSize(int corePoolSize) {
...
-
+        int corePoolSize = consumeExecutor.getCorePoolSize();
+        updateCorePoolSize(++corePoolSize);
     }
 
     @Override
     public void decCorePoolSize() {
-
+        int corePoolSize = consumeExecutor.getCorePoolSize();
+        updateCorePoolSize(--corePoolSize);
```
Comment (Git-Yang) [inline client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java:120]: Is it necessary to verify that corePoolSize is not less than or equal to 0?

**Comment by Git-Yang** [comment]
Can you create an issue for this PR?

**Comment by XiaoyiPeng** [comment]
> Can you create an issue for this PR?

OK, thanks for reminding me.

**Comment by guyinyou** [comment]
nice, LGTM!

**Comment by XiaoyiPeng** [comment]
> nice, LGTM!

Thanks for your review, can you approve these changes?

**Review by guyinyou — APPROVED** [review]

**Comment by tianliuliu** [comment]
LGTM

**Comment by Yelijah** [comment]
why this commit didn't be merged?

**Pull request closed without merging**